zulip_bots/zulip_bots/bots/ping/test_ping.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, List
+
+from zulip_bots.test_lib import BotTestCase
+
+class TestPingBot(BotTestCase):
+    bot_name = 'ping'
+
+    def get_responses(self, message: Dict[str, Any]) -> List[Dict[str, Any]]:
```

Comment:
If an equivalent method is not yet available in our bot testing library, can you add it there?